### PR TITLE
Enabling optional Timestamp fields tests

### PR DESCRIPTION
**Pull request by Abdiiir**
This is done to test against PR#46 --- Ensure that Timestamp doesn't fail on null on optional properties [#46](https://github.com/dcsaorg/DCSA-OVS/pull/46) ---

- testPortCallServiceTypeCodeField
- testModeOfTransportField
- testVesselPositionField
- testEventLocationField
- testEventLocationField
- testFacilitySMDGCodeField

- fix added to schema to allow nulling of vesselCallSignNumber. 

Note: This not expected to pass the test on OVS master

### CODE REVIEW

**Comment by nt-as** [comment]
Can we split this into two - the part that can be merged now and the parts pending a fix in OVS.

**Comment by Abdiiir** [comment]
> 
> 
> Can we split this into two - the part that can be merged now and the parts pending a fix in OVS.

Will do, waiting on a response.

**Review by douradofrazer — APPROVED** [review]

**Comment by Abdiiir** [comment]
This is deprecated as all tests on JITV1/OVS are enabled

**Pull request closed without merging**
